tests/Utility/HttpStatus/GetTextTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace WpOrg\Requests\Tests\Utility\HttpStatus;
+
+use WpOrg\Requests\Exception;
+use WpOrg\Requests\Exception\InvalidArgument;
+use WpOrg\Requests\Tests\TestCase;
+use WpOrg\Requests\Tests\TypeProviderHelper;
+use WpOrg\Requests\Utility\HttpStatus;
+
+/**
+ * @coversDefaultClass \WpOrg\Requests\Utility\HttpStatus
```

Comment:
```suggestion
 * @covers \WpOrg\Requests\Utility\HttpStatus::getText
```

As all tests in this test class are for the one method, there is no need for the individual test methods to have a `@covers` tag. A class level `@covers` tag will suffice.

(same for the other test class)